drawing/tools/custom/Axes.js

```diff
@@ -101,14 +101,14 @@ var Axes = oo.declare(
 
 			// NOTE: Not passing style into text because it's changing it
 			var props = {align:"middle", valign:"middle", util:this.util, annotation:true, container:this.container, mouse:this.mouse, stencil:this};
-			this.labelX = new Label(lang.mixin(props,{
+			this.labelX = new LabelExports.Label(lang.mixin(props,{
```

Comment:
Agreed.  I was following the name convention that Heng Liu had used in dojox/drawing/stencil/_Base.js.  I will go ahead and update both files.